policy/centos8/rke2.fc

```diff
@@ -6,6 +6,7 @@
 /usr/local/bin/rke2                                                 --  gen_context(system_u:object_r:container_runtime_exec_t,s0)
 /var/lib/cni(/.*)?                                                      gen_context(system_u:object_r:container_var_lib_t,s0)
 /var/lib/rancher/rke2(/.*)?                                             gen_context(system_u:object_r:container_var_lib_t,s0)
+/var/log/audit(/.*)?                                                    gen_context(system_u:object_r:container_var_lib_t,s0)
```

Comment:
this should already have a context that we don't want to replace?

Reply:
yeah, we cannot change this, we need to work with what is there:
```
[vagrant@node-1 ~]$ sudo ls -alZ /var/log/audit/
drwx------. root root system_u:object_r:auditd_log_t:s0 .
drwxr-xr-x. root root system_u:object_r:var_log_t:s0   ..
-rw-------. root root system_u:object_r:auditd_log_t:s0 audit.log
[vagrant@node-1 ~]$ 
```